src/main/scala/edu/ie3/util/scala/quantities/SpecificHeatCapacity.scala

```diff
@@ -25,15 +25,26 @@ final class SpecificHeatCapacity private (
 
   def dimension: SpecificHeatCapacity.type = SpecificHeatCapacity
 
-  def *(that: Temperature): EnergyDensity = WattHoursPerCubicMeter(
-    this.toWattHoursPerKelvinCubicMeters * that.toCelsiusScale
-  )
-  def multiply(temperature: Temperature, volume: Volume): Energy = WattHours(
-    this.toWattHoursPerKelvinCubicMeters * 1000 * temperature.toCelsiusScale * volume.toCubicMeters
-  )
+  def *(temperatureA: Temperature, temperatureB: Temperature): EnergyDensity =
+    KilowattHoursPerCubicMeter(
+      this.toKilowattHoursPerKelvinCubicMeters * math.abs(
+        temperatureA.toKelvinScale - temperatureB.toKelvinScale
+      )
+    )
 
-  private def toWattHoursPerKelvinCubicMeters: Double = to(
-    WattHoursPerKelvinCubicMeters
+  def multiply(
+      temperatureA: Temperature,
+      temperatureB: Temperature,
+      volume: Volume
+  ): Energy =
+    KilowattHours(
+      this.toKilowattHoursPerKelvinCubicMeters * math.abs(
+        temperatureA.toKelvinScale - temperatureB.toKelvinScale
+      ) * volume.toCubicMeters
+    )
+
+  def toKilowattHoursPerKelvinCubicMeters: Double = to(
+    KilowattHoursPerKelvinCubicMeters
   )
```

Comment:
Okay sorry for opening another issue:

I think we need some renaming here. There are to methods, one called multiply and one called * that one would generally expect to do the same thing. And for me both of them do something extra rather than just multiplying.

I would opt for:

1. Renaming of both to something like calcEnergy and calcEnergyDensity
2. Add some documentation to the functions as I'm for example not sure what temperatureA and temperatureB are.

More generally I would reserve the * operator only for simple multiplication like kw/^m2 * m^2  =kw